editor/src/templates/editor-entry-point.tsx

```diff
@@ -19,9 +19,10 @@ import { addStyleSheetToPage } from '../core/shared/dom-utils'
 import { STATIC_BASE_URL } from '../common/env-vars'
 
 const editorCSS = [
-  `${STATIC_BASE_URL}editor/css/canvas.css`,
-  `${STATIC_BASE_URL}editor/css/slider.css`,
-  `${STATIC_BASE_URL}editor/css/ReactContexify.css`,
+  `${STATIC_BASE_URL}editor/initial-load.css`,
+  `${STATIC_BASE_URL}editor/canvas.css`,
+  `${STATIC_BASE_URL}editor/slider.css`,
+  `${STATIC_BASE_URL}editor/ReactContexify.css`,
```

Comment:
agh, you tricked my eyes. you are pointing these to `editor/` but on master they point to `editor/css/`

Reply:
fixed!!